proxy/hdrs/HTTP.h

```diff
@@ -628,10 +629,12 @@ class HTTPHdr : public MIMEHdr
   const char *reason_get(int *length);
   void reason_set(const char *value, int length);
 
-  ParseResult parse_req(HTTPParser *parser, const char **start, const char *end, bool eof, bool strict_uri_parsing = false);
+  ParseResult parse_req(HTTPParser *parser, const char **start, const char *end, bool eof, bool strict_uri_parsing = false,
+                        size_t max_request_line_size = UINT16_MAX, size_t max_hdr_field_size = UINT16_MAX);
```

Comment:
Is the default value needs to be changed to 128K?

Reply:
Ah true. fixed it